### ISSUE TITLE
[Schedule Overrides] Ability for an override to last multiple days

### ISSUE THREAD

**Issue by kairandles** [netmix/radio-station#302]
**Is your feature request related to a problem? Please describe.**
Can you make where the end time is also give an optional field for end date

**Describe the solution you'd like**
I have a station that just plays non stop music during the school holiday's and I would like to just set a schedule overide for that time period

**Describe alternatives you've considered**
Currently, I am privating the regular shows then creating a show that is scheduled for every day 

**Additional context**
![image](https://user-images.githubusercontent.com/77554122/104824178-9aebd080-5847-11eb-9db2-1e9e0b829d32.png)

**Comment by majick777** [comment]
@kairandles I am going to merge this with #300 as they are a very similar issue.

**Comment by kairandles** [comment]
Hey, @majick777 just taking a quick look back at this and noticed that this doesn't really have that much to do with #300 as this is a feature request rather than a bug report? - Also I saw this hadn't been mentioned over there. Do you agree with the idea or?

**Comment by majick777** [comment]
@kairandles Yes on further consideration I'm going to re-open this, I see what you are getting at now.

**Comment by Paul1173** [comment]
And abillity to set more dates in 1 Schedule override

**Comment by kairandles** [comment]
@Paul1173 I've got wait I think you mean open on #301 already

**Comment by Paul1173** [comment]
sorry, you are right.

**Comment by majick777** [comment]
@kairandles @Paul1173 I've added the ability to add multiple Schedule Override times, so that a single Override can be assigned multiple times. That mostly solves this issue also as an Override Time could be added for each day (so long as full day overrides are working - will need to test this as per #300.) 
Maybe we can add an explicit end date field in the future, but it doesn't seem like a priority feature when there is this possibility here already.

**Comment by Paul1173** [comment]
Can't find option to set multiple override dates (only as show_override_date) but that is for a longer time a lot of work (ok, less than make an override schedule for each day..). There is also a show_override_sched, which value do we have to use?

**Comment by majick777** [comment]
@Paul1173 You'll need to download and install the development version (`develop` branch) for this, soon to be released as 2.3.3.9
The new version will convert `show_override_sched` to an array of single Override array values.
`show_override_date` is a separate index used only for sorting Overrides on the admin Override list.

**Comment by tonyzeoli** [comment]
@majick777  @kairandles  @Paul1173  Looks like this is resolved. Let's close this issue as it dates back 5 months or so. Any further needs on this, please open a new ticket.
